scaffolding/lib/utils.js

```diff
@@ -0,0 +1,240 @@
+const
+  fs = require('fs'),
+  ejs = require('ejs'),
+  _ = require('lodash'),
+  { exec } = require('child_process'),
+  path = require('path');
+
+function mkdirp (fullPath) {
+  const parts = fullPath.split(path.sep);
+
+  if (parts.length > 1) {
+    mkdirp(parts.slice(0, parts.length - 1).join('/'));
+  }
+
+  if (! fs.existsSync(fullPath)) {
+    fs.mkdirSync(fullPath);
+  }
+}
+
+function get(store, key) {
+  if (store && store[key]) {
+    return store[key];
+  }
+
+  return '';
```

Comment:
nitpicking: shouldn't this return `undefined` instead, to make sure we can distinguish any other value from that default one?

Reply:
Yes you're right. But this method isn't used any more, I removed it